client/packages/coldchain/src/Monitoring/ListView/TemperatureChart/BreachIcon.tsx

```diff
@@ -45,6 +45,7 @@ export const BreachIcon = ({
           startDateTime:
             DateUtils.getDateOrNull(payload.breach?.startDatetime) ??
             payload.date,
+          location: payload.breach?.location?.name,
```

Comment:
bit of a refactor here - instead of taking the location from the sensor, take it from the breach itself. 
More accurate and it removes the need to query the location for the sensor